sync.sh

```diff
@@ -51,7 +51,7 @@ $working_tree_root/init-tools.sh
 
 if [ "$sync_src" == true ]; then
     echo "Fetching git database from remote repos..."
-    git fetch --all -p -v &>> $sync_log
```

Comment:
@joperezr @maririos The `&` was there to redirect both stderr and stdout to the file.  Most of the output from `git fetch` comes from stderr, so it's not sufficient to just redirect stdout to the file.  [This documentation](http://www.tldp.org/LDP/abs/html/io-redirection.html) has more info about I/O redirection in Bash.

According to that guide, the `&>` was added in Bash 4, so that would explain the issue running it on OS X, where Bash 2 is still used by default, I think.  I had to make a fix for that reason in https://github.com/dotnet/corefx/commit/fe6ac97e4bf48c9a6ac675e8d567bf1ef2b9c92a.

It looks like a workaround is to use `>>filename 2>&1`; I'll test that and then send out a PR if it works.